backend/engine/plugins/trivy_sca/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Trivy SCA Scanner",
+    "type": "vulnerability",
+    "image": "$ECR/artemis/dind:latest",
+    "build_images": false,
+    "enabled": true
+}
```

Comment:
nit: Since this is a brand new file, let's go ahead and add this newline to the end of the file before it's merged to `main`